concepticondata/conceptlists/KBA-2021-912.tsv

```diff
@@ -0,0 +1,913 @@
+KBA-2021-912-427	427	m295-k-bp-fl-fa	to rake	3127	RAKE (SOMETHING)		ratisser, râteler	harken, rechen	hark	go haraka		4. Profession  4A. Agriculture	M269		
+KBA-2021-912-428	428	m296-k-bp-fl-fa	to plant	1486	PLANT (SOMETHING)		planter	pflanzen, anpflanzen	plant	go kabela,go lema		4. Profession  4A. Agriculture	M270		
+KBA-2021-912-429	429	m297-k-bp-fl-fa	to weed	3548	PULL UP WEEDS		désherber, arracher les mauvaises herbes de	jäten, Unkraut jäten	skoffel, onkruid uittrek	go tlhagola		4. Profession  4A. Agriculture	M271		
+KBA-2021-912-430	430	m298-k-bp-fl-fa	to harvest	1827	HARVEST CROPS		moissonner, récolter (cueillir)	ernten	oes			4. Profession  4A. Agriculture	M272		
+KBA-2021-912-431	431	m299-k-bp-fl-fa	to gather (fruits)	7	GATHER		recueillir, rassembler	sammeln	insamel	go kgetla		4. Profession  4A. Agriculture	M273		
+KBA-2021-912-432	432	m3-k91-bp29-fl-fa	man (male)	2106	MALE PERSON		homme	Mann	man	monna, ba-		1. Social life  1A.Social structure	M129	s094	
+KBA-2021-912-433	433	m30-k-bp-fl-fa	elder					Älteste(r)	oudste, ouderling	mogogi, ba-		1. Social life  1A.Social structure	M107		
+KBA-2021-912-434	434	m30-k94-bp-fl-fa	elder					Älteste(r)	oudste, ouderling	mogogi, ba-		1. Social life  1A.Social structure	M107		
+KBA-2021-912-435	435	m30-k95-bp-fl-fa	elder					Älteste(r)	oudste, ouderling	mogogi, ba-		1. Social life  1A.Social structure	M107		
+KBA-2021-912-436	436	m300-k-bp-fl-fa	to pick up	371	PICK UP		ramasser	auflesen, aufheben	ophef	go sela		4. Profession  4A. Agriculture	M274		
+KBA-2021-912-437	437	m301-k-bp-fl-fa	load	1679	LOAD		charge, chargement	Ladung, Last, Fracht	las, vrag	mopako		4. Profession  4A. Agriculture	M275		
+KBA-2021-912-438	438	m302-k-bp-fl-fa	stock (of grain)				réserve	Vorrat	voorraad (v. korrel)	mopako		4. Profession  4A. Agriculture	M276		
+KBA-2021-912-439	439	m303-k-bp-fl-fa	barn, granary				grenier (à blé), silo (à céréales)	Scheune, Speicher	skuur			4. Profession  4A. Agriculture	M277		
+KBA-2021-912-440	440	m304-k-bp-fl-fa	cattle	1153	CATTLE		bétail	Vieh	beeste	dikgomo		4B. Stock-farming	M278		
+KBA-2021-912-441	441	m305-k-bp-fl-fa	to rear	2134	RAISE (BRING UP)		élèver	halten, züchten	teel	go rua		4B. Stock-farming	M279		
+KBA-2021-912-442	442	m306-k-bp-fl-fa	to herd	2885	HERD (SOMETHING)			hüten	oppas, bewaak	go disa		4B. Stock-farming	M280		
+KBA-2021-912-443	443	m307-k-bp-fl-fa	bull	1008	BULL		taureau	Bulle, Stier	bul	poo		4B. Stock-farming	M281		
+KBA-2021-912-444	444	m308-k-bp-fl-fa	cow	1007	COW		vache	Kuh	koei	kgomo e namagadi		4B. Stock-farming	M282		
+KBA-2021-912-445	445	m309-k-bp58-fl-fa	to milk	93	DRAW MILK		traire	melken	melk	go gama		4B. Stock-farming	M283		
+KBA-2021-912-446	446	m31-k103-bp-fl-fa	friend	1325	FRIEND		ami	Freund	vriend	tsala, di-		1. Social life  1A.Social structure	M108		
+KBA-2021-912-447	447	m310-k-bp-fl-fa	to churn	696	STIR		baratter	buttern	botter maak, karring	go dira madila		4B. Stock-farming	M284		
```

Comment:
I would unmap this one since it refers specifically to the action of making butter, usually using a butter churn as a tool.